scripts/analyze.sh

```diff
@@ -248,6 +248,12 @@ fi
 
 if check_for_require fs || check_config_file ZJS_FS; then
     >&2 echo Using module: FS
+    OBJ_NAME=$(grep "require *( *['\"]fs['\"] *)*;" $SCRIPT | cut -d'=' -f1 | cut -d' ' -f2)
+
+    if grep "$OBJ_NAME\..*(" tests/test-fs-async.js | grep -q -v "Sync"; then
```

Comment:
Yes, I don't know how I didn't catch this. The test part is supposed to be $SCRIPT!